doc/contributing/adding-v8-fast-api.md

```diff
@@ -0,0 +1,147 @@
+# Adding V8 Fast API
+
+Node.js uses [V8](https://github.com/v8/v8) as the JavaScript engine.
```

Comment:
```suggestion
Node.js uses [V8](https://v8.dev/) as the JavaScript engine.
```
We should point to the official V8 project page instead of the GitHub mirror, as well as for consistency.